### PR TITLE
Modify S3334: Change text to LaYC (APPSEC-1219)

### PR DESCRIPTION
**Pull request by gaetan-ferry-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S3334/php/rule.adoc [hunk 1]
```diff
@@ -1,23 +1,52 @@
...
+; php.ini  Noncompliant; allow_url_fopen is enabled by default
 allow_url_include=1  ; Noncompliant
 ----
 
 
-=== Compliant solution
+==== Compliant solution
 
-[source,php]
+[source, php, diff-id=1, diff-type=compliant]
```
Comment (jamie-anderson-sonarsource) [inline rules/S3334/php/rule.adoc:49]:
```suggestion
[source,php,diff-id=1,diff-type=compliant]
```

**Review by jamie-anderson-sonarsource — CHANGES_REQUESTED** [review]
There are a few places where I think the wording could be changed to better meet a technical writing style.

**Comment by gaetan-ferry-sonarsource** [comment]
Thanks for the suggestion. I have been definitely too verbose here.

**Review by jamie-anderson-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**